src/java.xml/share/classes/module-info.java

```diff
@@ -229,16 +249,105 @@
+ * the {@link javax.xml.XMLConstants#FEATURE_SECURE_PROCESSING FEATURE_SECURE_PROCESSING}
+ * (hereafter referred to FSP), and the default values. The order of precedence
+ * for the configuration sources is defined as follows, with earlier ones overriding the later:
+ *
+ * <ul>
+ * <li><p>
+ *      Properties specified through factories or processors API;
+ * </li>
+ * <li><p>
+ *      Properties set using the corresponding System properties;
+ * </li>
+ * <li><p>
+ *      Properties set in a user-defined <a href="#ConfigurationFile">Configuration File</a>
+ * pointed to by the system property {@code java.xml.config.file};
+ * </li>
+ * <li><p>
+ *      Properties set in the default JAXP Configuration File
+ * <a href="#CF_Default">{@code jaxp.properties}</a>;
+ * </li>
+ * <li><p>
+ *      Properties' default values initialized during factory or processor creation.
+ * Security-related properties will be on their restrictive values when the
+ * {@link javax.xml.XMLConstants#FEATURE_SECURE_PROCESSING} feature is true.
+ * </li>
+ * </ul>
+ *
+ * Using the {@link javax.xml.catalog.CatalogFeatures CatalogFeatures}' RESOLVE
+ * property as an example, the followings illustrate how these rules are applied.
+ * <ul>
+ * <li><p>
+ *      Properties specified through factories or processors API have the highest
+ * precedence. The following code therefore effectively set the RESOLVE property
+ * to {@code strict}, regardless of settings in any other configuration sources.
+ * <pre>
+ *     {@code
+ *     DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+ *     dbf.setAttribute(CatalogFeatures.Feature.RESOLVE.getPropertyName(), "strict");
+ *     }
+ * </pre>
+ *
+ * </li>
+ * <li><p>
+ *      If the property is not set on the factory such as in the above code, a
+ * system property setting will be in effect.
+ * <pre>
+ *     {@code
+ *     // in the following example, the RESOLVE property is set to 'continue'
+ *     // for the entire application
+ *     java -Djavax.xml.catalog.resolve=continue myApp
+ *
+ *     // in the following code snipet, the property is set to 'ignore' for the
+ *     // factory that follows
+ *     System.setProperty("javax.xml.catalog.resolve", "ignore");
+ *     DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+ *     ...
+ *     System.clearProperty("javax.xml.catalog.resolve");
+ *     }
+ * </pre>
+ * </li>
+ * <li><p>
+ *      If the property is not set on the factory, nor through its system property,
+ * the setting in a configuration file will take effect. The following entry
+ * sets the property to '{@code continue}'.
+ * <pre>
+ *     {@code
+ *     javax.xml.catalog.resolve=continue
+ *     }
+ * </pre>
+ * </li>
+ * <li><p>
+ *     If the property is not set anywhere, it will be resolved to its
+ * default value that is '{@code strict}'.
+ * </li>
+ * </ul>
+ *
  * @implNote
+ *
+ * <h2 id="ConfigurationFile">JAXP Configuration File</h2>
```

Comment:
I thin this means there are two anchors with the same name and two sections in the module description with the same title. I understand this is SE vs. JDK specific properties but this could easily go over the headers of someone reading this. I think it would be better if "Implementation Specific Features and Properties" were the first section of the implNote. It can start by saying that the JAXP Configuration file (link to the description at the top) can also be used for JDK/implementation specific properties.

Reply:
Removed. Incorporated with the sub-section "configuration file" within the "Implementation Specific Features and Properties" section.